addons/stock_account/models/stock.py

```diff
@@ -419,12 +419,14 @@ def _action_done(self):
         for move in res.filtered(lambda m: m.product_id.valuation == 'real_time' and (m._is_in() or m._is_out() or m._is_dropshipped() or m._is_dropshipped_returned())):
             move._account_entry_move()
 
+        max_moves_to_vacuum = int(self.env['ir.config_parameter'].sudo().get_param('stock_account.max_moves_to_vacuum'))
```

Comment:
If parameter doesn't exist int(False) -> 0 Then search with limit=0 will not returns anymove if the param doesn't exist

Reply:
That's what I thought too, but after some tests, I came out that `limit=0` is returning all the moves.

Reply:
limit is not added to the query when its value is 0.
https://github.com/odoo/odoo/blob/7938a1a58e714dc8c6cae4b384fb5d5a7e231931/odoo/models.py#L4162